import-service/.env

```diff
@@ -0,0 +1,5 @@
+DB_MASTER_USERNAME=yankouskia
```

Comment:
? :)

Reply:
I bet some crawlers already parsed this file =))

Reply:
lol :D thanks for paying attention